vm/adb/adb.go

```diff
@@ -221,7 +221,12 @@ func (inst *instance) repair() error {
 	// Ignore errors because all other adb commands hang as well
 	// and the binary can already be on the device.
 	inst.adb("push", inst.cfg.Executor, "/data/syz-executor")
```

Comment:
Please also remove this command.
And the following piece in executor/executor.cc:
```
	if (argc == 2 && strcmp(argv[1], "reboot") == 0) {
		reboot(LINUX_REBOOT_CMD_RESTART);
		return 0;
	}
```